accounts/forms.py

```diff
@@ -334,6 +334,8 @@ class Meta:
         model = Profile
         fields = ('home_page', 'about', 'signature', 'sound_signature', 'is_adult', 'not_shown_in_online_users_list', )
 
+    def get_img_fields(self):
```

Comment:
I'd be nice to document this as this function as just by looking at the form class it won't be obvious to understand why this is here. If I'm not wrong the fields returned are the `HtmlCleaningCharField` fields, therefore you might want to rename the method to something like `get_html_cleaning_fields`.

Reply:
In fact, maybe you can refactor the function so that it dynamically checks the type of the fields and returns them? (so that we don't have to update it if we ever add a new `HtmlCleaningCharField`.

Reply:
The thing is that field being of `HtmlCleaningCharFields` class doesn't mean it needs img notifications (there are examples of title fields in other modules)